src/screens/Feed/FeedScreen.tsx

```diff
@@ -20,7 +20,7 @@ const FeedScreen = ({ navigation }: any) => {
       setList(data);
     };
     void fetchJobs();
-  }, []);
+  }, [list]);
```

Comment:
was adding "list" in the [] necessary to update the feed screen?

Reply:
it's necessary to reload the feed screen every time the list is changed so that the 'like' buttons are up to date